layouts/partials/label-sandbox.html

```diff
@@ -0,0 +1,8 @@
+<span class="pf-c-label pf-m-orange">
+  <span class="pf-c-label__content">
+    <span class="pf-c-label__icon">
+      <i class="fas fa-fw fa-circle-exclamation" aria-hidden="true"></i>
```

Comment:
This is just me but when I see an exclamation alongside a term, I read it as a warning and to tread with caution. Is that our intention here?